src/test/testSchemaGenerator.ts

```diff
@@ -2062,51 +2062,55 @@ describe('attachDirectiveResolvers on field', () => {
 
   const directiveResolvers: IDirectiveResolvers<any, any> = {
     lower(
-      next: NextResolverFn,
+      next: NextResolverFn<any>,
       src: any,
+      inputArgs: { [argName: string]: any },
```

Comment:
I think we need to make this work in a way where existing usages don't change, otherwise it's a pretty major break. Can we make it optional to pass the stuff into `next`?

Reply:
I agree! I'll change it.